webapp/src/cljs/lipas/ui/map/utils.cljs

```diff
@@ -497,6 +498,16 @@
       (utils/round-safe 2)
       read-string))
 
+(defn calculate-area
+  [fcoll]
+  (-> fcoll
+      clj->js
+      turf-area
+      ->clj
+      (convertArea "meters" "kilometers") 
```

Comment:
For "future us" I think we could refactor the function names to include the output unit:
- `calculate-area` -> `calculate-area-km2`
- `calculate-route-length` -> `calculate-route-length-km`

Otherwise we need to guess the output unit or read the source. It's weird btw that `turf/length` returns kilometres by default. I could've sworn it would return meters and I had to check their api-docs. Maybe this would be a good place for a comment.

What do you think?

Reply:
Renamed with return type and added a clarifying comment.